build/bk-build-utils.js

```diff
@@ -96,18 +98,76 @@
       return path.extname(file) === '.go';
     });
 
+    if (!platformSupportsPlugins()) {
+      pluginsToInclude.push({
+        name: pluginName,
+        path: pluginPath,
+        files: goFiles
+      });
+      return Q.resolve();
+    }
+
     var args = ['build', '-i', '-buildmode=plugin', '-o', pluginName + '.so'];
     args = args.concat(goFiles);
     return spawnProcess('go', args, pluginPath, env);
   }
 
-  function build(path, exeName) {
+  function build(srcPath, exeName) {
     var args = ['build', '-i', '-o', exeName];
-    return spawnProcess('go', args, path, env);
+
+    if (!platformSupportsPlugins()) {
+      prepareBuildWithoutPluginSupport(srcPath);
+    }
+
+    return spawnProcess('go', args, srcPath, env);
   }
 
   function test(path) {
     return spawnProcess('go', ['test', '-v'], path, env);
   }
 
+  function platformSupportsPlugins() {
+    var isWin = /^win/.test(process.platform);
+    return !isWin;
```

Comment:
Think we should check if platform is linux rather than not windows. That would cover Mac OS as well.